### ISSUE TITLE
Allow integration with third parties on dojo page

### ISSUE THREAD

**Issue by Wardormeur** [CoderDojo/community-platform#1145]
Depending on what data-sources you provide on your dojo page, integration could be done in a way it displays the latest informations from those sources.
As an example : 
- if you provide a twitter account, it should display the latest tweets
- if you provide a tumblr account, it should display the latest images

Those informations should be toggable when providing those datasource so that you either display it as a link or as integrated
From https://forums.coderdojo.com/topic/310/uploading-images-to-dojo-listings-on-zen
Thoughts @rosalanghammer @DanielBrierton ?

**Comment by rosalanghammer** [comment]
She means more of a gallery. I imagine our original idea for cover photo equivalent would work. 

On a separate note - yes twitter feeds etc would be good but we would need to think about where this would go as it could get cluttered fast if you had facebook and twitter even - a toggle on and off would be good. Also distracts from events.

**Comment by Wardormeur** [comment]
The goal of the integration was to avoid having to do a gallery : image upload is sensitive in multiple way, and you won't have enough feature to cover everything that may be needed; hence a better integration with a social network who's image-oriented would make more sense.
Another solution would to allow markdown from nodebb down to the profile page, but "somebody" would still have to host the image.

**Comment by rosalanghammer** [comment]
Yeah good point. Then I assume the obvious integration would be flickr, which lots of Dojos use, as it's solely photo sharing - there may be another I don't know about. Also like the idea of feeds but it could get messy, if we want to add just for the gallery then I imagine using a photo sharing platform would be the best way to approach.

**Comment by AT1452** [comment]
Is this issue still up for grabs? If so I believe I can help
